packages/app/src/fields.tsx

```diff
@@ -35,43 +34,48 @@ const FieldError = ({ message }: { message?: string }) => {
   )
 }
 
-export const TextField = (props: Pick<InputProps, 'size' | 'autoFocus' | 'secureTextEntry'>) => {
+export const TextField = ({
+  autoCapitalize = 'none',
+  spellCheck = false,
+  autoCorrect = false,
+}: Pick<InputProps, 'autoCapitalize' | 'spellCheck' | 'autoCorrect'>) => {
   const {
     field,
     error,
     formState: { isSubmitting },
   } = useTsController<string>()
-  const { label, placeholder, isOptional, maxLength, isEmail, isURL, uniqueId } =
-    useStringFieldInfo()
+  const { label, placeholder, isOptional, maxLength } = useStringFieldInfo()
+
   const themeName = useThemeName()
-  const id = useId()
   const disabled = isSubmitting
-  const isPrettyName = !isURL && !isEmail && uniqueId !== 'path'
+
   return (
     <Theme name={error ? 'red' : themeName} forceClassName>
       <Fieldset>
         {!!label && (
-          <Label theme="alt1" size={props.size || '$3'} htmlFor={id}>
```

Comment:
I don't think we need this "htmlFor" at this point and `useId()` does not work for some reason very well within ts-form. It produces duplicate IDs. From my research, this might be related to useId() when used inside a fragment.

Reply:
oh hmm. Well the `htmlFor` is needed so the correct input gets focused when you tap on the label.